lib/analyzer/applications/java.ts

```diff
@@ -3,40 +3,34 @@ import * as path from "path";
 import { bufferToSha1 } from "../../buffer-utils";
 import { JarFingerprintsFact } from "../../facts";
 import { JarFingerprint } from "../types";
-import { JarBuffer, JarCoords, JarInfo } from "./types";
+import { AggregatedJars, JarBuffer, JarCoords, JarInfo } from "./types";
 import { AppDepsScanResultWithoutTarget, FilePathToBuffer } from "./types";
 
+/**
+ * @param {{[fileName: string]: Buffer}} input fileName
+ * @returns {AggregatedJars}
+ */
 function groupJarFingerprintsByPath(input: {
```

Comment:
Hmm what do you think about having a more meaningful name than `input`? How about something like `fileNameToBuffer`?

Reply:
Changes